packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -6421,7 +6445,6 @@ export default class Meeting extends StatelessWebexPlugin {
       this.statsAnalyzer = new StatsAnalyzer({
         // @ts-ignore - config coming from registerPlugin
         config: this.config.stats,
-        receiveSlotCallback: (ssrc: number) => this.receiveSlotManager.findReceiveSlotBySsrc(ssrc),
```

Comment:
Why are we removing this? Is this not needed for meetings also ?

Reply:
@Kesari3008 this logic has been updated, so now we don't have to have `receiveSlotCallback` here.

You can see how it works now here:

WCME PR here: https://sqbu-github.cisco.com/CPaaS/web-client-media-engine/pull/392
internal-media-core PR here: https://sqbu-github.cisco.com/WebExSquared/webrtc-media-core/pull/234